README.md

```diff
@@ -332,6 +332,7 @@ gordalina_mixpanel:
     enabled: true                                 # defaults to true
     enable_profiler: %kernel.debug%               # defaults to %kernel.debug%
     auto_update_user: %kernel.debug%              # defaults to %kernel.debug%
+    user_data_display_errors: %kernel.debug%      # defaults to %kernel.debug%
```

Comment:
Missing renaming this field and the PR is good to go.

Reply:
oups indeed, thanks